Tests/IGListSingleSectionControllerTests.m

```diff
@@ -75,14 +75,14 @@ - (void)test_whenItemUpdated_thatCellIsConfigured {
                              genTestObject(@3, @"Baz"),
                              ]];
     self.dataSource.objects = @[
-                                genTestObject(@1, @"Foo"),
-                                genTestObject(@2, @"Qux"), // new value
+                                genTestObject(@1, @"Qux"), // new value
+                                genTestObject(@2, @"Bar"),
                                 genTestObject(@3, @"Baz"),
```

Comment:
I changed this so that the first object is updated instead of the second, because on tvOS `cellForItemAtIndexPath` doesn't return a cell for the second section, presumably because it's out of range.

Reply:
> because on tvOS cellForItemAtIndexPath doesn't return a cell for the second section, presumably because it's out of range.

Ummm this seems weird, it should return the right section data if the indexPath is correct, can you investigate a bit more why the test doesn't work for tvOS?

Reply:
@koenpunt did u figure out why this has to change?

Reply:
Not yet, I'll dive into this a bit more tomorrow.

Reply:
The behavior of `cellForItemAtIndexPath` is just different for tvOS. When I change the frame of the collection view to be bigger than 100x100, even only slightly (100x105), it works. I yet have to see how this behaves when running in the tvOS simulator, but maybe that will make it clear why this is happening.